playbooks/app_dev/builds.adoc

```diff
@@ -335,9 +337,13 @@ There are a number of existing S2I builders that you can look to leverage for yo
 
 === Binary deployments
 
-In certain cases, an application may be previously compiled outside of an OpenShift build. You can use the _assemble_ phase of an S2I build process to retrieve the previously packaged artifact and insert it into the typical build process of the builder being leveraged.
 
-The following is an example of a deployment of a binary application for the OpenShift JBoss EAP image
+In certain cases, an application may be previously compiled outside of an OpenShift build. There are two methods to include an existing binary artifact(s) in the resulting image:
+
+1. Customize the _assemble_ script as part of the S2I build process to retrieve a previously packaged artifact from a remote source (such as an artifact repository) and place in the JBoss deployments folder.
```

Comment:
The content here is generic to any language that can be compiled/built. What I would maybe recommend is more clearly separate the content into

- Generic guidance about how a binary deployment might be done (no refs to JBoss sepcifically)
- A working example using JBoss EAP